pkg/tinydtls/Makefile.include

```diff
@@ -2,7 +2,7 @@ PKG_BUILDDIR ?= $(PKGDIRBASE)/tinydtls
 
 INCLUDES += -I$(PKG_BUILDDIR)
 
-ifeq ($(shell uname -s),Darwin)
+ifneq (,$(filter $(TOOLCHAIN),llvm))
```

Comment:
Could you replace this with
ifeq ($(TOOLCHAIN), llvm) 
You don't need filter since it's not a list.